doc/response-tagging.rst

```diff
@@ -1,17 +1,17 @@
-Extra Invalidation Handlers
-===========================
+Response Tagging
+================
 
-This library provides decorators that build on top of the ``CacheInvalidator``
-to simplify common operations.
+This library provides a service to handle tagging of responses ``ResponseTagger``
+to simplify use of tags.
 
 .. _tags:
 
-Tag Handler
------------
+Response Tagger
+---------------
```

Comment:
i don't think we need two titles for response tagging. this document was about all handlers, but there never was more than 1 so i agree it makes more sense this way. but one title and introduction is enough